lib/iris/tests/unit/fileformats/pp/test_save.py

```diff
@@ -44,6 +44,18 @@ def test_grid_and_pole__scalar_dim_longitude(unit, modulus):
     assert field.lbnpt == lon.points.size
 
 
+def test_realization():
+    cube = stock.lat_lon_cube()
+    real_coord = DimCoord(42, standard_name="realization", units=1)
+    cube.add_aux_coord(real_coord)
+    with mock.patch("iris.fileformats.pp.PPField3", autospec=True) as pp_field:
+        pp_field.lbrsvd = list(range(6))
```

Comment:
I think I understand that this is necessary, unlike the other examples using `verify()`, because we want to assign a member of `lbrsvd` within this `Mock`, whereas the others are directly setting a property because those are single-number fields. Makes sense to me.

However: Iris is currently only aware of `lbrsvd` being 4 numbers long:

https://github.com/SciTools/iris/blob/3ad940f906772836cd12e06e056b921600c73835/lib/iris/fileformats/pp.py#L165

So we should keep the pattern consistent here for minimum confusion:

```suggestion
        pp_field.lbrsvd = list(range(4))
```